combinations/hash.tsv

```diff
@@ -195,3 +195,5 @@ bioconductor-diffbind=3.4.0,bioconductor-csaw=1.28.0,bioconductor-edger=3.36.0,r
 r-shazam=1.1.0,r-tigger=1.0.0
 bamcmp=2.2	quay.io/bioconda/base-glibc-debian-bash:latest	0
 ascat=3.0.0,cancerit-allelecount=4.3.0
+biscuit=1.0.2.20220113,samtools=1.14
```

Comment:
why is this container needed if you have the samblaster one as well?

Reply:
It's not strictly required. However, I anticipate that some folks will not prefer the samblaster pipeline; I was hoping to minimize requirements for that case.

Reply:
Happy to just keep the 3-tool container if you think that's best

Reply:
It would save storage and cost at our side.

Reply:
Sounds good. Removed the redundant line